src/main/java/com/tictactoe/Program.java

```diff
@@ -10,12 +10,22 @@
     private GameSettings menu = new GameSettings(receiver, validator);
     private GamePreference preference = menu.collectGamePreference();
 
-    private Player player1 = new EasyComputerPlayer (preference.player1Symbol, board);
-    private Player player2 = new EasyComputerPlayer (preference.player2Symbol, board);
+    private Player player1 = preference.player1;
+    private Player player2 = preference.player2;
 
-    private Game tictactoe = new Game(player1, player2, board);
+    private String player1Symbol = preference.player1Symbol;
+    private String player2Symbol = preference.player2Symbol;
+
+    private Game tictactoe = new Game();
 
     public void run() {
```

Comment:
I like this `run` method a lot. It is very expressive - I can see, step by step, how the game is set up